### DISCUSSION TITLE
"Notch-tick" sound effect spammed when dragging while settings are open in pause menu

### DISCUSSION THREAD

**Discussion by eggooooo** [ppy/osu#15714] [Q&A] [answered]
In the settings overlay when pausing gameplay & dragging an adjustment notch, the "notch-tick" sound is being played more than it should (or you could call it being "spammed".

This only happens when the game is paused and only affects the "scroll speed" afaik, could be that this occurs in some other places too.

Video explaining the issue in further detail here:

https://user-images.githubusercontent.com/64386789/142757300-0813f564-eb83-4dfe-8ee7-8a29e86ca3bf.mp4

**Comment by bdach** [comment, marked as answer]
#15718
